CONTRIBUTING.md

```diff
@@ -405,6 +407,29 @@ README for getting the docs tooling set up.
 node scripts/docs.js --open
 ```
 
+### Release Notes Process
+
+Kibana publishes major, minor and patch releases periodically through the year. During this process we run a script against this repo to collect the applicable PRs against that release and generate [Release Notes](https://www.elastic.co/guide/en/kibana/current/release-notes.html). To include your change in the Release Notes:
+
+1. In the title, summarize what the PR accomplishes in language that is meaningful to the user.  In general, use present tense (for example, Adds, Fixes) in sentence case.
+1. Label the PR with the targeted version (ex: 6.5).
+1. Label the PR with the appropriate github labels:
```

Comment:
In Contributing.md, the section titled "Getting to the Code Review Stage" includes this line:

Do not assign a version label. Someone from Elastic staff will assign a version label, if necessary, when your Pull Request is ready to be merged.

We need to be consistent.

Reply:
Just a note here: Community contributors cannot place ANY labels to anything on GitHub. So we should maybe make this more clear in the documentation, that all the labelling will be done by a maintainer, and this sections only applies to maintainers.